File: Leetcode/Javascript/217-ContainsDuplicate.js
const nums = [1, 2, 3];
var containsDuplicate = function (nums) {
  let temp = 0;

  for (let i = 0; i < nums.length; i++) {
    temp = nums[i];
    for (let j = i + 1; j < nums.length; j++) {
      if (nums[j] === temp) {
        return true;
      }
    }
  }
  return false;
};
console.log(containsDuplicate(nums));

File: Leetcode/217-ContainsDuplicate.cpp
#include <iostream>
#include <set>
#include <vector>

class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        set<int> bucket;
        for(int i =0;i<nums.size();i++){
            bucket.insert(nums[i]);
        };
        
        if (bucket.size() == nums.size()) {
            return false;
        } else {
            return true;
        }
    }
};

int main() {
}
